.github/workflows/pwa.yml

```diff
@@ -0,0 +1,107 @@
+name: PWA deploy
```

Comment:
This was used to test this PWA in a production environment. You can check it out [here](https://sunflower-land.com/pwa)